### PR TITLE
Add network and session bus permission

### PR DESCRIPTION
**Pull request by kevinsmia1939**
Attempt to fix https://github.com/flathub/org.pymol.PyMOL/issues/21

### CODE REVIEW

File: org.pymol.PyMOL.yaml [hunk 1]
```diff
@@ -7,7 +7,8 @@ finish-args:
   - --share=ipc
   - --socket=x11
   - --device=dri
-
+  - --share=network
+  - --socket=session-bus
```
Comment (hfiguiere) [inline org.pymol.PyMOL.yaml:11]: You should actually not allow the session-bus but rather the specific talk names.

Comment (kevinsmia1939) [reply org.pymol.PyMOL.yaml:11]: How do I know what talk name it will be?

Comment (hfiguiere) [reply org.pymol.PyMOL.yaml:11]: `--log-session-bus` passed to flatpak run.

Comment (hfiguiere) [reply org.pymol.PyMOL.yaml:11]: but I didn't get the dbus error here.

Comment (kevinsmia1939) [reply org.pymol.PyMOL.yaml]: I did not get the dbus error too, I will just give it network permission.

**Pull request merged**